cpp/clang/readme.md

```diff
@@ -0,0 +1,16 @@
+# How to setup clang in your project
+To begin with, these configs are based on clang tools 17+ versions, so in order to work with it you should update your clang tools (For installation example in docker you can refer to [embedded-tools](https://github.com/emlid/embedded-tools/blob/pr-journal-and-filesystem/docker/Dockerfile)) project).
```

Comment:
I think we should leave here a link to the `dev`, not `pr` branch. Because now it's in branch only, but then you merge it and delete a branch, and there would be nothing and we would need to change this link.

Reply:
dev is empty now. I made a link just to show that it already works and planned to change it later